src/main/waveform/waveform.core.js

```diff
@@ -66,7 +66,7 @@
    * data) or 'json'
    * @param {HTMLMediaElement} options.mediaElement
    *
-   * @see Refer to the <a href="https://github.com/bbc/audiowaveform/blob/master/doc/DataFormat.md">data format documentation</a>
+   * @see Refer to the <a href="../../../doc/DataFormat.md">data format documentation</a>
```

Comment:
This link is to a [doc](https://github.com/bbc/audiowaveform/blob/master) in the audiowaveform repo.